src/main/kotlin/org/bundleproject/json/assets/ModData.kt

```diff
@@ -0,0 +1,18 @@
+package org.bundleproject.json.assets
+
+import com.google.gson.annotations.SerializedName
+
+enum class ModSource {
```

Comment:
Bit of a nitpick and not really required to complete but rather just an experiment, could we use sealed classes instead 🤔 ?